playbooks/provisioner/openstack/main.yaml

```diff
@@ -4,3 +4,5 @@
   roles:
       - { role: openstack/neutron, action: setup }
       - { role: openstack/nova, action: setup }
+
+- include: plugins.yml
```

Comment:
why "plugins"? it's too vague (we already have 2 uses for plugins in this project) can you please rename it otherwise?